### PR TITLE
Add d2l-table.patch

### PR DESCRIPTION
**Pull request by awikkerink**

### CODE REVIEW

File: patches/d2l-table.patch [hunk 1]
```diff
@@ -0,0 +1,189 @@
+diff --git a/d2l-tspan.js b/d2l-tspan.js
+index 47a863e..0e9282e 100644
+--- a/d2l-tspan.js
++++ b/d2l-tspan.js
+@@ -1,6 +1,6 @@
+ import '../@polymer/polymer/polymer-legacy.js';
+ import './d2l-table-observer-behavior.js';
+-import '../resize-observer-polyfill/resize-observer.js';
++import ResizeObserver from '../resize-observer-polyfill/dist/ResizeObserver.es.js';
```
Comment (svanherk) [inline patches/d2l-table.patch]: Will this need to be updated everywhere the resize observer is used?  Should we add it to the script?

File: patches/d2l-table.patch [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+-			vars: {
+-				overridePath: 'd2l-table'
+-			}
+-		});
+-	}
+-});
+diff --git a/test/d2l-table.html b/test/d2l-table.html
+index dfd90eb..80f178e 100644
+--- a/test/d2l-table.html
++++ b/test/d2l-table.html
```
Comment (svanherk) [inline patches/d2l-table.patch]: Minor: Should we just make this change on master/hybrid?

File: patches/d2l-table.patch (hunk 1 above)
Comment (awikkerink) [reply patches/d2l-table.patch]: Yes, the resize-observer.js file doesn't exist. Probably a good idea.

File: patches/d2l-table.patch (hunk 2 above)
Comment (awikkerink) [reply patches/d2l-table.patch]: Sure.

File: patches/d2l-table.patch (hunk 1 above)
Comment (svanherk) [reply patches/d2l-table.patch]: You should be able to remove this from the patch now since we added this to the script.

**Pull request merged**